srcpkgs/freerdp3/template

```diff
@@ -19,7 +19,8 @@ configure_args="-DCHANNEL_URBDRC_CLIENT=ON
  -DWITH_FFMPEG=$(vopt_if ffmpeg ON OFF) -DWITH_PCSC=$(vopt_if smartcard ON OFF)
  -DWITH_PKCS11=$(vopt_if smartcard ON OFF) -DWITH_SWSCALE=$(vopt_if ffmpeg ON OFF)
  -DWITH_WAYLAND=$(vopt_if wayland ON OFF) -DWITH_WEBVIEW=$(vopt_if webview ON OFF)
- -DWITH_CLIENT_SDL_VERSIONED=$(vopt_if sdl2 ON OFF)"
+ -DWITH_CLIENT_SDL_VERSIONED=$(vopt_if sdl2 ON OFF) -DWITH_VERBOSE_WINPR_ASSERT=OFF
+ -DWITH_INTERNAL_MD4=ON -DWITH_INTERNAL_RC4=ON"
```

Comment:
Why did you add `-DWITH_INTERNAL_MD4=ON -DWITH_INTERNAL_RC4=ON` here.

From my understanding MD4/MD5/RC4 are provided by OpenSSL and support for them can be configured by the user by enabling/disabling legacy crypto in the OpenSSL config. (Also, from my understanding those should only be needed for NTLM, whereas Kerberos supports newer crypto)

https://github.com/FreeRDP/FreeRDP/issues/10908#issuecomment-2517338121

Reply:
these warnings (and an error) are produced IF `configure_args` above are not enabled:
```
[ERROR][com.winpr.crypto.hash] - [winpr_Digest_Init_Internal]: Failed to initialize digest md4
[WARN][com.freerdp.core.rdp] - [log_build_warn_hash][0x55f7c955a3c0]: *************************************************
[WARN][com.freerdp.core.rdp] - [log_build_warn_hash][0x55f7c955a3c0]: [SSL] {Digest} build or configuration missing:
[WARN][com.freerdp.core.rdp] - [log_build_warn_hash][0x55f7c955a3c0]:  * md4: NTLM support not available
[WARN][com.freerdp.core.rdp] - [log_build_warn_hash][0x55f7c955a3c0]: *************************************************
[WARN][com.freerdp.core.rdp] - [log_build_warn_cipher][0x55f7c955a3c0]: *************************************************
[WARN][com.freerdp.core.rdp] - [log_build_warn_cipher][0x55f7c955a3c0]: [SSL] {Cipher} build or configuration missing:
[WARN][com.freerdp.core.rdp] - [log_build_warn_cipher][0x55f7c955a3c0]: * rc4: assistance files with encrypted passwords, NTLM, RDP licensing and RDP security will not work
[WARN][com.freerdp.core.rdp] - [log_build_warn_cipher][0x55f7c955a3c0]: *************************************************
```

Reply:
What command are you running to produce those errors?

Reply:
Alright, those warnings are printed in `freerdp_context_new()`, whereas `winpr_InitializeSSL()` doesn't get called until `freerdp_connect()`.

https://github.com/FreeRDP/FreeRDP/blob/6e24bdd817f6aeae8d2a0e0d19606934e9588b2d/libfreerdp/core/freerdp.c#L805
https://github.com/FreeRDP/FreeRDP/blob/6e24bdd817f6aeae8d2a0e0d19606934e9588b2d/libfreerdp/core/connection.c#L321

Reply:
I brought this up on the matrix and a patch has been submitted upstream by the maintainer to resolve this: https://github.com/FreeRDP/FreeRDP/pull/11023

Reply:
added the patch.